modules/custom/express_content_edit_lock/express_content_edit_lock.module

```diff
@@ -80,11 +81,7 @@ function express_content_edit_lock_admin_form_submit($form, &$form_state) {
   global $user;
   // Exclude unnecessary elements.
   form_state_values_clean($form_state);
-  $allowed_users = explode(',', $form_state['values']['lock_allow_users']);
-
-  foreach ($allowed_users as $user) {
-    trim($user);
-  }
+  $allowed_users = explode(', ', $form_state['values']['lock_allow_users']);
 
   $form_state['values']['lock_allow_users'] = $allowed_users;
```

Comment:
This seems weird that a string is turned into an array and inserted back into the form values.